src/Column/ActionColumn.php

```diff
@@ -0,0 +1,458 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\DataView\Column;
+
+use Closure;
+use InvalidArgumentException;
+use Yiisoft\Html\Tag\A;
+use Yiisoft\Html\Tag\Span;
+use Yiisoft\Router\UrlGeneratorInterface;
+
+use function is_array;
+
+/**
+ * ActionColumn is a column for the {@see GridView} widget that displays buttons for viewing and manipulating the items.
+ */
+final class ActionColumn extends Column
+{
+    private array $buttons = [];
+    private string $primaryKey = 'id';
+    private string $template = '{view}{update}{delete}';
+    private array $urlArguments = [];
+    private Closure|null $urlCreator = null;
+    private bool $urlEnabledArguments = true;
```

Comment:
```suggestion
    private bool $urlEnableArguments = true;
```

Doesn't `$urlArguments = []` disable it already?

Reply:
done.

Reply:
done.